web/simonamarcu2/html/primul/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+      <!DOCTYPE html>
```

Comment:
please remove the empty spaces from Line 1

Reply:
fixed, thank you!